src/components/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -129,13 +129,16 @@ export const ContributionsEpicButtons = ({
                 <>
                     <PrimaryCtaButton cta={cta} tracking={tracking} countryCode={countryCode} />
 
-                    {secondaryCta && secondaryCta.baseUrl && secondaryCta.text ? (
+                    {secondaryCta?.type === SecondaryCtaType.Custom &&
+                    secondaryCta.cta.baseUrl &&
+                    secondaryCta.cta.text ? (
```

Comment:
are `baseUrl` and `text` guaranteed to exist at this point?

Reply:
Oh there's actually a bit of a type mismatch between admin-console and SDC here. In the admin-console cta type is: 
```ts
export interface Cta {
  text?: string;
  baseUrl?: string;
}
```

but in SDC it's
```ts
export interface Cta {
    text: string;
    baseUrl: string;
}
```

We should probably get these to align! I would say the SDC  is the way to go (i.e. these fields **shouldn't** be optional - may be worth doing in a follow up?

Reply:
And to actually answer the question - SDC thinks they're guarenteed at this point, but technically they aren't given the types in the tool

Reply:
Oh I see. Yes I agree that both fields should be mandatory in the tool as well